calico_node/filesystem/etc/calico/confd/templates/bird_ipam.cfg.template

```diff
@@ -2,20 +2,34 @@
 filter calico_pools {
   calico_aggr();
   custom_filters();
-{{range ls "/pool"}}{{$data := json (getv (printf "/pool/%s" .))}}
+{{range ls "/v1/ipam/v4/pool"}}{{$data := json (getv (printf "/v1/ipam/v4/pool/%s" .))}}
   if ( net ~ {{$data.cidr}} ) then {
     accept;
   }
 {{end}}
   reject;
 }
 
+{{$network_key := printf "/bgp/v1/host/%s/network_v4" (getenv "NODENAME")}}{{$network := getv $network_key}}
 filter calico_ipip {
-{{range ls "/pool"}}{{$data := json (getv (printf "/pool/%s" .))}}{{if $data.ipip}}
+{{range ls "/v1/ipam/v4/pool"}}{{$data := json (getv (printf "/v1/ipam/v4/pool/%s" .))}}{{if $data.ipip}}
   if ( net ~ {{$data.cidr}} ) then {
+{{if $network}}
+    krt_tunnel = "{{$data.ipip}}";
+    accept;
+  }
+{{else if $data.ipip_off_subnet}}
+    if ( gw ~ {{$network}} ) then {
```

Comment:
Hi Rob - I couldn't get this filter to work - I ended up using "gw != <gateway address of subnet>" instead - BIRD complained about a type conversion? (e.g. from an address to a CIDR) What does the "network_v4" key contain in this version?